chart/values.yaml

```diff
@@ -7,28 +7,26 @@
 ## @param replicaCount Number of replicas
 replicaCount: 1
 
-## @param image.digest Image digest in the way sha256:aa...
 ## @param image.pullPolicy image pull policy
 ## @param image.pullSecret Name of secret used to pull image
 ## @param image.registry Image registry
-## @param image.repository Image repository
+## @param image.namespace Image registry namespace
+## @param image.name Name of the image in the registry
 ## @param image.tag Image tag
 ##
 image:
-  digest: ""
   pullPolicy: IfNotPresent
   pullSecret: ""
-  registry: ""
-  repository: ""
-  # Overrides the image tag whose default is the chart appVersion.
+  registry: "ghcr.io"
+  namespace: "ucl-mirsg"
+  name: "xnat-core"
```

Comment:
I don't think we should set these as defaults. At least the `name` should be left as `""`.

Reply:
yeah good point, I'll make `name` an empty string